code/src/components/Library.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { BASE_URL } from 'utils/urls';
+
+const Library = () => {
+  const [library, setLibrary] = useState([]);
+
+  useEffect(() => {
+    fetch(BASE_URL)
+      .then((response) => response.json())
+      .then((data) => setLibrary(data.results))
+      .catch((error) => console.error(error))
+  }, []);
+
+  return (
+    <section className="library">
+      {library.map((movie) => {
+        return (
+          <div className="release-container">
+            <Link
+              key={movie.id}
+              to={`/details/${movie.id}`}
+              className="movie">
+              <div className="img-overlay" />
+              <img src={`https://image.tmdb.org/t/p/w342/${movie.poster_path}`} alt="poster" />
```

Comment:
This image size could might have been bigger? The pictures are a bit blurry when looking at the page on desktop.